common/ownership_voucher/pdc_pub.pem

```diff
@@ -0,0 +1,32 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I suggest to move non-code files to testdata folder

Reply:
I've moved it to a testdata subdirectory. I'd like to keep it here as the Go embed directive only works in the current directory.